software/base/src/test/java/org/apache/brooklyn/entity/software/base/VanillaSoftwareProcessIntegrationTest.java

```diff
@@ -51,7 +51,7 @@ public void setUp() throws Exception {
         super.setUp();
         localhost = app.getManagementContext().getLocationRegistry().getLocationManaged("localhost");
         
-        runRecord = Files.createTempFile("testVanillaSoftwareProcess-runRecord", ".txt");
+        runRecord = Files.createTempFile("testVanillaSoftwareProcess-runRecord-"+Identifiers.makeRandomId(8), ".txt");
```

Comment:
Not sure what guarantees `createTempFile` gives. If the previous temp files have been deleted, then might it reuse the name? Particularly if we run two tests within the same millisecond, which admittedly seems unlikely when they are integration tests.